src/olympia/files/models.py

```diff
@@ -706,6 +712,50 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
+    KNOWN_PERMISSIONS = WEBEXT_PERMISSIONS_DICT
+    name = models.CharField(max_length=255, unique=True, null=False)
+    _enum = models.IntegerField(default=WEBEXT_UNKNOWN.id, db_column='enum')
```

Comment:
fair point - I started with the constants and enum before I'd finished the model.  
We can't do choices alas - there will be permissions we don't yet know about (e.g. a Chrome one; a new one that's just landed) and also urls for match patterns that I think we should keep.